### PR TITLE
Update Playwright for Login.gov and Accessibility Testing

### PR DESCRIPTION
**Pull request by JCantu248**
#  #

## 🗣 Description ##

This PR adds axe-core for accessibility testing of Crossfeed website. As part of the UI testing, Playwright will scan pages for accessibility issues. Also includes fixes for logging into Crossfeed through the new Login.gov portal in order to develop new features.

## 💭 Motivation and context ##

This change adds UI accessibility testing to the CI/CD pipeline for Crossfeed.

## 🧪 Testing ##

Changes were tested locally and as part of the CI/CD pipeline on push.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [ ] Revert dependencies to default branches.
- [ ] Finalize version.

## ✅ Post-merge checklist ##

- [ ] Create a release.

### CODE REVIEW

**Review by schmelz21 — APPROVED** [review]
LGTM
Looks like we need some support on this review @dav3r and @jsf9k. Thanks

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -13,10 +13,6 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: mcr.microsoft.com/playwright:v1.41.2-jammy
-    # TODO: Refactor action to use updated logic for Login.gov
-    # when a service account to run tests is available for use.
-    # Issue: CRASH-337
-    if: true == false
```
Comment (dav3r) [inline .github/workflows/playwright.yml:19]: It's unclear to me why this comment is being removed- can someone please explain?

Comment (JCantu248) [reply .github/workflows/playwright.yml:19]: A service account has been provided. We needed a test account/mailbox so the testing scripts in Github Actions would not be using an individual user account. We added the new account credentials in Github Secrets and the TODO is resolved.

Comment (dav3r) [reply .github/workflows/playwright.yml:19]: Great, thanks for the clarification @JCantu248. 👍

**Review by dav3r — APPROVED** [review]
Approved with regard to the changes in `.github/workflows/playwright.yml`.  I can't speak to rest of these changes, I'll leave that up to the XFD team.

Generally, I'd recommend splitting up the playwright testing changes from the "fixes for logging into Crossfeed through the new Login.gov portal" so that they are in two separate PRs (because they don't appear to be particularly related).  Since they are combined into this PR, I'd at least recommend updating the name of this PR to mention those fixes.

**Pull request merged**